test/createStore.spec.ts

```diff
@@ -519,10 +520,10 @@ describe('createStore', () => {
 
   it('does not throw if action type is falsy', () => {
     const store = createStore(reducers.todos)
-    expect(() => store.dispatch({ type: false })).not.toThrow()
-    expect(() => store.dispatch({ type: 0 })).not.toThrow()
+    expect(() => store.dispatch({ type: false } as any)).not.toThrow()
```

Comment:
Again, because the `store` is properly typed, TS won't compile this, so I'm just casting `as any` to allow it to compile. Pretty common to see `as any` usage in tests. In fact, I usually change my `.eslintrc` to allow `any` in test files.